src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Runtime
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic), Shared]
+    public class PreferStreamAsyncMemoryOverloadsFixer : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds =>
+            ImmutableArray.Create(PreferStreamAsyncMemoryOverloads.RuleId);
+
+        public sealed override FixAllProvider GetFixAllProvider() =>
+            WellKnownFixAllProviders.BatchFixer;
+
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            Document doc = context.Document;
+            CancellationToken ct = context.CancellationToken;
+            SyntaxNode root = await doc.GetSyntaxRootAsync(ct).ConfigureAwait(false);
+            if (root.FindNode(context.Span) is SyntaxNode node)
```

Comment:
I think you need to specify `getInnermostNodeForTie = true` for case where invocation itself is an argument to some other invocation. Good to validate with a unit test.

```suggestion
            if (root.FindNode(context.Span, getInnermostNodeForTie: true) is SyntaxNode node)
```

Reply:
Pending adding a unit test.

Reply:
Unit test added, and it passed! Please take a look.